components/eam/bld/configure

```diff
@@ -1752,6 +1752,12 @@ if (defined $opts{'cppdefs'}) {
 }
 $cfg_ref->set('cppdefs', $usr_cppdefs);
 
+# define is_scream_config flag to select which P3 version codes to use
+my $is_scream_config = 0;
+if ($usr_cppdefs =~ /\-DSCREAM/) {
```

Comment:
Sanity check question for @jgfouca or @bartgol or someone in the know - does -DSCREAM in cppdefs mean that the simulation is either a SCREAM F90 or a SCREAM C++ run, or does it just imply one of these 2 options?

Reply:
Good catch!

IIRC, that's used only by Homme. But I _think_ we never tested it for SCREAMv0, so I am not sure what it would produce there. Grepping in Homme folder, all but a few of the matches are of the form
```
#if !defined(CAM) && !defined(SCREAM)
```
or its opposite without the `!`'s. There are only a few cases where CAM and SCREAM would be treated differently:
- adjust_ps is `false` for SCREAM, and `true` otherwise
- for theta-l_kokkos target, if SCREAM is defined, forcing is _not_ pushed back to f90 (I guess we assume we're running eamxx), while we do push to f90 for CAM.

I suspect the last of the two points above would cause wrong results in this case. I would leave `-DSCREAM` out of the compile line, and use `-DCAM` instead.

Reply:
Yeah, I have vague recollection that `DSCREAM` does not mean what a normal person would think. Maybe we should create a git issue in SCREAM to figure out whether we still need it or not? Though maybe it will naturally go away when we remove support for the F90 code in a few months?

Reply:
EAMxx _does_ need `-DSCREAM` (and in fact we do have it). But I think v0 should be considered as "EAM" from the point of view of Homme.